spec/features/hq/create_sponsor_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.feature 'User enters new sponsor data', :type => :feature do
+
+  background do
+    i_sign_in_as_admin
+    an_agent_exists_with_username
+  end
+
+  scenario 'User clicks the New Sponsor button' do
+    when_i_register_new_sponsor
+
+    then_the_sponsor_should_be_saved
+    and_i_should_be_on :new_sponsor_page
+  end
+
+  scenario 'With invalid input' do
+    visit new_hq_sponsor_path
+    click_button 'Create and Add Another'
+    expect(page).to have_content 'Please fix the errors below and resubmit the form.'
+  end
+
+  def when_i_register_new_sponsor
+    FactoryGirl.create(:sponsor, city: 'Sigurdland')
+    visit new_hq_sponsor_path
```

Comment:
instead of going directly to new sponsor page, better go to sponsor index page and click New Sponsor button so we also that that button.
(no need to text the left navigation bar here because there is a different task for that)